package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/worksheet.bundle.js",
   "type": "module",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha.js --require ./tests/preload.js ./tests/*",
+    "test": "./node_modules/mocha/bin/mocha.js --require --timeout 3000 ./tests/preload.js ./tests/*",
```

Comment:
This will not work for running the tests. You need to put `--require` in front of the preload location

Reply:
weird that it __does__ work